src/World.cc

```diff
@@ -900,6 +943,16 @@ sdf::ElementPtr World::ToElement(const OutputConfig &_config) const
         this->dataPtr->sphericalCoordinates->ElevationReference());
     sphericalElem->GetElement("heading_deg")->Set(
         this->dataPtr->sphericalCoordinates->HeadingOffset().Degree());
+    if (sphericalElem->HasElement("surface_axis_equatorial"))
```

Comment:
I would remove this `HasElement` check to maintain consistency with the other `ToElement` functions, and then add corresponding expectations to the `ToElement` test

Reply:
The surface axes tags are only used with custom surfaces and would be ignored for EARTH_WGS84 when passed as sdf, but theres no harm in keeping them in the ``ToElement()`` method, just to be consistent.

Reply:
> I had suggested adding the HasElement check since these fields aren't required and may not have been originally provided.

Sorry I didn't read the full comment history before adding my own.

> Then the output of ToElement()->ToString("") would be:

Yes, there is an open issue (https://github.com/gazebosim/sdformat/issues/867) about this exact issue. I believe the rationale for the current behavior is to allow DOM objects to be constructed and configured via the programmatic API without an SDFormat file or string.

Reply:
So should I remove the ``HasElement()`` check ?

Reply:
I think it is more consistent with the existing behavior to remove the `HasElement` checks

Reply:
Done https://github.com/gazebosim/sdformat/pull/1050/commits/a74dd1068eb903f33a7c6611051380cb36cb8d2a